cylc/flow/workflow_db_mgr.py

```diff
@@ -440,7 +440,8 @@ def put_task_pool(self, pool: 'TaskPool') -> None:
         relevant insert statements for the current tasks in the pool.
         """
         self.db_deletes_map[self.TABLE_TASK_POOL].append({})
-        self.db_deletes_map[self.TABLE_TASK_PREREQUISITES].append({})
+        # We now keep prereq history:
```

Comment:
Might be a bit clearer
```suggestion
        # We now keep prereq history, no need to do:
```

Reply:
Done.